snaplet-sqlite-simple.cabal

```diff
@@ -1,5 +1,5 @@
 name:           snaplet-sqlite-simple
-version:        0.4.6
+version:        0.6.0.1
```

Comment:
so, I noticed snaplet-postgresql-simple moved to 1.0.0.0 from 0.6.0.4 to resemble update to support snap 1.0, should we do it same way?